rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -422,12 +426,13 @@ def _sample_episodes(
         num_items: Optional[int] = None,
         *,
         batch_size_B: Optional[int] = None,
-        batch_length_T: Optional[int] = None,
+        batch_length_T: Optional[int] = 0,
```

Comment:
Let's leave the default value at `None`.
It makes more sense semantically for cases, where no sequences are needed.